src/components/new-request/name-check/name-check.vue

```diff
@@ -0,0 +1,809 @@
+<template>
+  <v-container id="name-check-main-container" class="white pa-0 rounded">
+    <name-check-issues-dialog :attach="'#app'"
+                              :display="showNameCheckIssuesDialog"
+                              :options="dialogOptions"
+                              @proceed="dialogSubmit"/>
```

Comment:
I like to see parentheses on the method (just to be explicit and easier to recognize).

Reply:
sure